src/layout-grid.scss

```diff
@@ -144,11 +144,18 @@ $cols: 12;
       margin-top: 0;
     }
 
-    &--no-gap {
+    &--no-horizontal-gap {
```

Comment:
maybe `.row--no-gap` is more suitable (similar to .row-no-gutters in Bootstrap)

Reply:
As discussed, we can't due to negative margines